go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -737,13 +729,17 @@ func (bnc *BaseNetworkController) getPodNADNames(pod *kapi.Pod) []string {
 	return podNadNames
 }
 
+func (bnc *BaseNetworkController) getClusterPortGroupDbIDs(base string) *libovsdbops.DbObjectIDs {
+	return libovsdbops.NewDbObjectIDs(libovsdbops.PortGroupCluster, bnc.controllerName,
+		map[libovsdbops.ExternalIDKey]string{
+			libovsdbops.ObjectNameKey: base,
+		})
+}
+
 // getClusterPortGroupName gets network scoped port group hash name; base is either
 // ClusterPortGroupNameBase or ClusterRtrPortGroupNameBase.
 func (bnc *BaseNetworkController) getClusterPortGroupName(base string) string {
-	if bnc.IsSecondary() {
-		return libovsdbutil.HashedPortGroup(bnc.GetNetworkName()) + "_" + base
```

Comment:
this old logic is irrelevant because indexing allows for multiple controllers correct?